biz.aQute.bndlib.tests/test/test/model/EETest.java

```diff
@@ -143,6 +145,17 @@ public void checkEEHasModules(EE ee) throws Exception {
 		assertThat(ee.getModules()).isNotEmpty();
 	}
 
+	@Test
+	public void testEEIsJDKPackage() {
```

Comment:
Should also have an equivalent test for this in `Analyzer` to check (eg) `javax.xml.transform.stax` against JavaSE-1.5 (which should complain about missing version ranges) and JavaSE-1.6 (which shouldn't).

Reply:
done in https://github.com/bndtools/bnd/pull/6508/commits/2cd573d423e3fff52238f9bcbd06bfc033cc629e